apps/condo/domains/miniapp/schema/AppMessageSetting.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * Generated by `createschema miniapp.AppMessageSettings 'app?:Relationship:B2CApp:CASCADE; blockReason:Text; blockedNotificationTypes:Text; throttlingTime:Integer;'`
+ */
+
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const { OPERATION_FORBIDDEN } = require('@condo/domains/common/constants/errors')
+const access = require('@condo/domains/miniapp/access/AppMessageSetting')
+const { getMessageTypeField } = require('@condo/domains/notification/schema/fields/MessageType')
+
+
+const ERRORS = {
+    APP_MESSAGE_SETTING_MUST_HAVE_ONLY_B2B_OR_B2C_APP_FIELD: {
+        code: BAD_USER_INPUT,
+        type: OPERATION_FORBIDDEN,
+        message: 'AppMessageSetting must have either b2bApp or b2cApp field',
+    },
+}
+
+const AppMessageSetting = new GQLListSchema('AppMessageSetting', {
+    schemaDoc: 'Configuration of sending push notifications for specific miniapps.' +
+        'Used to block certain types of push notifications and control throttling of notifications of a certain miniapp',
+    fields: {
+
+        b2bApp: {
+            schemaDoc: 'Link to B2BApp to which this config applies',
+            type: 'Relationship',
+            ref: 'B2BApp',
+            isRequired: false,
+            kmigratorOptions: { null: true, on_delete: 'models.CASCADE' },
+            access: {
+                update: false,
+            },
+        },
+
+        b2cApp: {
+            schemaDoc: 'Link to B2CApp to which this config applies',
+            type: 'Relationship',
+            ref: 'B2CApp',
+            isRequired: false,
+            kmigratorOptions: { null: true, on_delete: 'models.CASCADE' },
+            access: {
+                update: false,
+            },
+        },
+
+        blockReason: {
+            schemaDoc: 'Reason for blocking messages with types from type',
+            type: 'Text',
+        },
+
+        isBlacklisted: {
```

Comment:
Can I just blacklist specific notification by setting `numberOfNotificationInWindow` to `0`? :)